src/main/kotlin/com/team4099/robot2020/config/ControlBoard.kt

```diff
@@ -25,4 +25,10 @@ object ControlBoard {
 
     val wristHorizontal: Boolean
         get() = operator.rightShoulderButton
+
+    val runIntakeIn: Boolean
+        get() = operator.aButton
```

Comment:
Although not that important right now, you should start to consider how to make the controls easier for the operator. For example, if I were to press A, it would both move the wrist down and run the intake inwards.

Reply:
This isn't a specific problem to this subsystem though. The correct way to handle this would be through a state machine or superstructure (we likely need an intake/feeder state machine anyway, so the wrist automatically moving can be rolled into that).

Reply:
Yeah thats why I said that its not that important right now, and only as a thing to consider for ease of controls.